src/Symfony/Component/Intl/Locales.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Intl;
+
+use Symfony\Component\Intl\Exception\MissingResourceException;
+
+/**
+ * Gives access to locale-related ICU data.
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+final class Locales extends AbstractStaticBundle
+{
+    /**
+     * @return string[]
+     */
+    public static function getCodes(): array
```

Comment:
```suggestion
    public static function getLocales(): array
```

The values returned here are commonly refered to as "locale". I don't think it is a good idea to abstract this away with the ambiguous concept of "code".